### ISSUE TITLE
Handlers GenericProviderDispatcher and MustUnderstandValidationDispatcher not available

### ISSUE THREAD

**Issue by tigran-a** [holodeck-b2b/Holodeck-B2B#137]
Hi, 

Classes org.apache.axis2.jaxws.dispatchers.GenericProviderDispatcher and org.apache.axis2.jaxws.dispatchers.MustUnderstandValidationDispatcher used in the list of handlers are not shipped with the release. Can we safely remove those handlers from the Dispatch phase? 

https://github.com/holodeck-b2b/Holodeck-B2B/blob/f892b5366cb8b74d3bd4d0bc564ea61e35a72941/modules/holodeckb2b-distribution/basedir/conf/holodeckb2b.xml#L214

Thank you

**Comment by RenateS** [comment]
Yes, these can be removed as they are not used. We will do so too in the next release.
We are wondering if this issue causes trouble running Holodeck B2B? As have never heard of trouble in runtime caused by this.

**Comment by tigran-a** [comment]
Thank you for the answer. 
No, it doesn't cause troubles, just cause WARN messages "unable to load... " in the log
